### PR TITLE
[fixed] Handle case when no tabbable element exists

### PR DESCRIPTION
**Pull request by evoyy**
Fixes https://github.com/rackt/react-modal/issues/44

### CODE REVIEW

**Comment by claydiffrient** [comment]
@evoyy This looks like a great change.  Can you add a spec that tests the functionality?  I generally like specs to accompany code changes.

**Comment by evoyy** [comment]
@claydiffrient Spec added.

File: specs/Modal.spec.js [hunk 1]
```diff
@@ -92,6 +92,12 @@ describe('Modal', function () {
     });
   });
 
+  it('handles case when child has no tabbable elements', function() {
```
Comment (claydiffrient) [inline specs/Modal.spec.js]: It apperas that you forgot to make an assertion in the spec. It just opens the modal, presses tab then closes it.

Comment (evoyy) [inline specs/Modal.spec.js]: That's intentional. The spec simply verifies that an exception is not thrown when a tab is pressed. Without the bugfix, the spec will fail.

Comment (evoyy) [inline specs/Modal.spec.js]: @claydiffrient I have updated the spec to use assert.doesNotThrow().

**Comment by claydiffrient** [comment]
Okay, I checked out the code and took a look at the actual functionality this makes happen.  I'm slightly concerned because this fix allows focus to escape from the modal, which from an accessibility standpoint isn't ideal.  Granted I think most use cases of the modal will include a close button of some nature, which avoids this problem.  This is nice in that it avoids the errors being thrown when there is nothing to tab to when the user tabs like is currently on master.

I think this is good for now.  I want to think about maybe adding a way to switch between modal and non-modal in the future probably though.

**Pull request merged**